specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -392,6 +392,149 @@
           }
         ]
       }
+    },
+    "/providers/Microsoft.Subscription/aliases/{aliasName}": {
+      "put": {
+        "description": "Create Alias Subscription.",
+        "operationId": "Subscription_CreateAlias",
+        "x-ms-examples": {
+          "CreateAlias": {
+            "$ref": "./examples/CreateAlias.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Normal response for a successful query.",
+            "schema": {
+              "$ref": "#/definitions/PutAliasResponse"
+            }
+          },
+          "202": {
```

Comment:
This operation needs to be marked as x-ms-long-running-operation if its async.  https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation

Reply:
Updated